src/shared/util/getTodayDate.ts

```diff
@@ -0,0 +1,10 @@
+export const getTodayDate = () => {
+  const now = new Date();
+  const year = now.getFullYear();
+  const month = String(now.getMonth() + 1).padStart(2, "0");
+  const day = String(now.getDate()).padStart(2, "0");
+  const hours = String(now.getHours()).padStart(2, "0");
+  const minutes = String(now.getMinutes()).padStart(2, "0");
+
+  return `${year}-${month}-${day}T${hours}:${minutes}`;
+};
```

Comment:
Since you have installed `date-fns`, you can use its format method to achieve more shortly.